keyboards/system76/launch_1/readme.md

```diff
@@ -0,0 +1,99 @@
+# System76 Launch Configurable Keyboard (launch_1)
+
+![System76 Launch Configurable Keyboard](https://images.prismic.io/system76/b71307ac-dae6-4863-b7ca-804cd61c7ef8_launch_overhead.png?auto=compress,format&w=750)
+
+The Launch Configurable Keyboard is engineered to be comfortable, fully customizable, and make your workflow more efficient.
+
+- High-speed USB Hub
+- Works on Linux, Windows and macOS
+- 100% Open Source
+- Made in Colorado
+
+Additional Launch Keyboard resources:
+
+- Keyboard Maintainer: [System76](https://github.com/system76)
+- Hardware Supported: [System76 Launch GitHub Repository](https://github.com/system76/launch)
+- Hardware Availability: [Shop System76](https://system76.com/accessories/launch)
+
+## Prerequisites
+
+Refer to [build environment setup](https://beta.docs.qmk.fm/tutorial/newbs_getting_started) instructions.
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+- **Bootmagic reset**: Hold down the key at (0,0) in the matrix (i.e., Escape) and plug in the keyboard
+- **Keycode in layout**: Press the key mapped to `RESET` if it is available
+<!-- - **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead -->
+
+## Building Firmware
+
+Start with QMK's [Complete Newbs Guide](https://beta.docs.qmk.fm/tutorial/newbs).
+
+In summary:
```

Comment:
Again, already documented. Please just stick to the readme template.

Reply:
I have followed the template more closely. I hope it is to your liking.